tests/misc/CachingTest.py

```diff
@@ -146,13 +146,11 @@ def test_caching_multi_results(self):
                 retval, output = execute_coala(
                    coala.main,
                    'coala',
-                   '-c', filename + '.coafile',
-                   '-f', filename + 'test.py')
+                   '-c', filename + '.coafile')
```

Comment:
@jayvdb I fixed it in https://github.com/coala/coala/pull/3178/commits/f1c1b30e12a1b543cf631e964ab469003694c525. It was needed here because it needed to be in the default section. In the later commit I merge them for proper deprecation and hence fixed it back. `coala -f filename` is needed in case a section doesn't have a files argument.